### PR TITLE
Closes #1887: Addressing some nits from the find in page feature component that were pending in #1858.

### LINKED ISSUE #1887: Follow-up fix nits on find in page feature component
There were some nits that need to be addressed, related https://github.com/mozilla-mobile/android-components/pull/1858

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features
- [ ] ~~**Changelog**: This PR includes a changelog entry or does not need one~~
**No need it, this PR just rewording some text to make it clear.**

### CODE REVIEW

File: samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt [hunk 1]
```diff
@@ -120,7 +120,7 @@ open class DefaultComponents(private val applicationContext: Context) {
         )
     }
 
-    var onFindPageClicked: (() -> Unit)? = null
+    var onFindInPageClicked: (() -> Unit)? = null
```
Comment (Amejia481) [inline samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt]: @csadilek Do you want to use this PR for chatting about this lambda?

File: components/feature/findinpage/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ implementation "org.mozilla.components:feature-findinpage:{latest-version}"
 ```
 
 ### Adding feature find in page.
-To use feature find in page you have to do two things:
+To use this feature have to do two things:
```
Comment (csadilek) [inline components/feature/findinpage/README.md]: nit: To use this feature *you*, or just "Using this feature requires:"

File: components/feature/findinpage/src/main/java/mozilla/components/feature/findinpage/FindInPageView.kt [hunk 3]
```diff
@@ -39,10 +38,10 @@ class FindInPageView @JvmOverloads constructor(
     var onCloseButtonPressListener: (() -> Unit)? = null
 
     /**
-     * Gives access to the underneath engine, to start the search and to control the navigation between
+     * Gives access to the underneath sessionEngine to start the search and to control the navigation between
```
Comment (csadilek) [inline components/feature/findinpage/src/main/java/mozilla/components/feature/findinpage/FindInPageView.kt]: nit: Comment still references `sessionEngine`. So, should be "..access to the engine session to start...."

File: samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt (hunk 1 above)
Comment (csadilek) [reply samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt]: @Amejia481 let's talk through options when you're back. I don't have a quick solution that I like.

Comment (Amejia481) [reply samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt]: 👍

Comment (pocmo) [reply samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt]: I did it a bit differently when integrating this into the ref browser (see PR). It's still not super nice but it avoids leaking UI by attaching it to the components object.

**Pull request merged**